hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/LocalArtifactRepository.java

```diff
@@ -56,19 +56,19 @@
     List<Artifact> findBySha1Hash(String sha1Hash);
 
     /**
-     * Verifies if an artifact exists that has given hash and is still related
-     * to a {@link SoftwareModule} other than a given one and not
-     * {@link SoftwareModule#isDeleted()}.
+     * Verifies if in the same tenant one then more {@link Artifact}s with the
```

Comment:
Needs to be adapted:
`    /**
     * Counts current elements based on the sha1 and tenant, as well as having
     * the {@link SoftwareModule} property 'deleted' with value 'false'
     * 
     * @param sha1
     *            the sha1 of the {@link Artifact}
     * @param tenant
     *            the current tenant
     * 
     * @return the count of the elements
     */
`